src/sdk/python/src/iotea/core/util/json_query.py

```diff
@@ -15,11 +15,12 @@
 # :<label> > Gets value as is
 # foo.bar:<label> > Gets nested value 'baz' in { foo: { bar: 'baz' }}
 # *.bar:<label> > Gets all nested values 'baz1' and 'baz2' in { foo1: { bar: 'baz1' }, foo2: { bar: 'baz2' }}
-# foo[1]:<label> > Gets second member of array in { foo: [0, 2, 3] } = 2
-# foo[1:2]:<label> > Gets range of array in { foo: [0, 2, 3] } > 2, 3
+# foo[1]:<label> > Gets second member of array in { foo: [0, 2, 3] } > 2
+# foo[1:2]:<label> > Gets range of array in { foo: [0, 2, 3] } > 2
```

Comment:
Good catch 👍 but should the comment then not more like _foo[1:3] ... > [2, 3]_

Reply:
It should show, that the last item is excluded in the range selector. We could add your example as well in the list.